src/fprime/common/models/serialize/numerical_types.py

```diff
@@ -44,6 +44,7 @@ def getSize(cls):
     @abc.abstractmethod
     def get_serialize_format():
         """Gets the format serialization string such that the class can be serialized via struct"""
+        raise NotImplementedError("get_serialize_format")
```

Comment:
Is this a part of the build cache changes?

Reply:
No, it fixes an outstanding static analysis issue.  This method shouldn't be called (overridden in all implementations) so it shouldn't harm to properly raise this exception.